dcos-log/mesos/files/reader/read_test.go

```diff
@@ -21,9 +21,14 @@ five
 `)
 )
 
-func createHandler(data []byte, t *testing.T) http.HandlerFunc {
+func createHandler(data []byte, useResponse bool, t *testing.T) http.HandlerFunc {
```

Comment:
i might rename `useResponse` here to `forceSendResponse` just to make the intent clear.